docs/onvif-configuration.md

```diff
@@ -91,8 +91,8 @@ By default, the generic ONVIF Configuration will create services for all the bro
 
 ## Modifying a Configuration
 More information about how to modify an installed Configuration, add additional protocol Configurations to a cluster, or
-delete a Configuration can be found in the [Modifying a Akri Installation
-document](./modifying-akri-installation.md).
+delete a Configuration can be found in the [Customizing Akri Installation
```

Comment:
@kate-goldenring @bfjelds , should all these by Modifying _an_ Akri installation and if so, should I rename the docs to match?